src/Sylius/Bundle/AdminBundle/Resources/views/ProductVariant/_form.html.twig

```diff
@@ -1,24 +1,18 @@
-{% set form = hookable_data.form %}
-{% set main_hook = hook_name(hookable_data.parent_main_hook, 'form') %}
-{% set fallback_hook = hook_name(hookable_data.parent_fallback_hook, 'form') %}
-
+{% if form is not defined %}
+    {% set form = hookable_metadata.context.form %}
+{% endif %}
 {% form_theme form '@SyliusAdmin/Shared/form_theme.html.twig' %}
 
 <div class="container-xl">
     {{ form_start(form, {'attr': {'class': 'ui loadable form', 'novalidate': 'novalidate', 'id': form.vars.id}}) }}
     <div class="row">
-        {% if hookable_configuration.method is defined %}
-            <input type="hidden" name="_method" value="{{ hookable_configuration.method }}" />
+        {% if hookable_metadata.configuration.method is defined %}
+            <input type="hidden" name="_method" value="{{ hookable_metadata.configuration.method }}" />
         {% endif %}
         {{ form_errors(form, sylius_test_form_attribute('form-validation-errors')) }}
         {{ form_widget(form._token) }}
 
-        {% hook [main_hook, fallback_hook] with {
-            parent_main_hook: main_hook,
-            parent_fallback_hook: fallback_hook,
-            resource: hookable_data.resource,
-            form,
-        } %}
+        {% hook 'form' %}
```

Comment:
Maybe
```suggestion
        {% hook 'form' with { form }%}
```
just so we're consistent about the usage 🤷

Reply:
Yeah, I've added it somewhere as it was required (probably it was causing some magic with `form_theme` but for some reason it wasn't complaining in other places).

I've added it everywhere "just in case", as it's harmless change.